common/app/routes/challenges/utils.js

```diff
@@ -239,6 +239,15 @@ export function getCurrentSuperBlockName(current, entities) {
   return block.superBlock;
 }
 
+export function challengeTitle({ blockName, title }) {
+  if ((!blockName || blockName === 'undefined') && title) {
```

Comment:
This is from an earlier iteration, I think I was using `params` at the time, the string of 'undefined' was being passed to the updateTitle action, So I had to account for it. Removing this shouldn't cause an issue now though.